### PR TITLE
Dev louis settings

### PR DESCRIPTION
**Pull request by lindlou**
Add image and video formats
Add rotations and flips
Remember parameters from previous session (brighness, path etc)
Add startup message in terminal (version, authors, license)
Add missing shortcuts for mouseless operation

### CODE REVIEW

File: rpyscope/gui_qt.py [hunk 1]
```diff
@@ -216,7 +223,6 @@ def __init__(self):
 
         self.path_button = QPushButton("Browse")
         self.path_button.clicked.connect(self.set_path)
-        self.path_button.acceptDrops()
```
Comment (trappitsch) [inline rpyscope/gui_qt.py:219]: Why is this removed? I think it allows a user to select a path by simply dragging a file onto the button... might be good to check.

File: rpyscope/gui_qt.py [hunk 2]
```diff
@@ -36,7 +37,16 @@ class MainWindowControls(QMainWindow):
     def __init__(self):
         # info variables
         self.version = "0.0.1"
-        self.author = "Reto Trappitsch"
```
Comment (trappitsch) [inline rpyscope/gui_qt.py:39]: This should also be edited in `setup.py`. Alternatively - and maybe better - we could kick it out of here (since I don't think it is used anywhere) and create an `AUTHORS.md` file in the repo. I don't have strong feelings about this (at this point) either way.

File: rpyscope/gui_qt.py [hunk 3]
```diff
@@ -509,35 +603,39 @@ def set_path(self):
             self.path_input.setText(str(path))
 
     def reset_resolution(self):
-        self.res_input.setText(self.config.get("default_resolution"))
+        self.res_input.setText(self.config._get_default("resolution"))
 
     def set_resolution(self):
         new_res = self.res_input.text()
-        if self.cam.resolution != new_res:
+        if str(self.cam.resolution) != new_res:
```
Comment (trappitsch) [inline rpyscope/gui_qt.py:610]: Why turn this into a `str`, shouldn't it always be a string?

**Review by trappitsch — CHANGES_REQUESTED** [review]
Looks pretty good. I have a general comment that is outside this PR that i will put in an issue.

File: rpyscope/gui_qt.py (hunk 1 above)
Comment (lindlou) [reply rpyscope/gui_qt.py:219]: acceptDrops tells the system that the push button may be able to accept drop events, but I didn't write any function to handle this event.  I don't know why I wrote this line in the first place. To me this looks unnecessary, but if you really like it I can implement it, but it makes more sense to me to drag onto the text input.

File: rpyscope/gui_qt.py (hunk 3 above)
Comment (lindlou) [reply rpyscope/gui_qt.py:610]: Before I turned it into `str`, they were never equal. This is because `type(self.cam.resolution)` returns `picamera.mmalobj.PiResolution`

File: rpyscope/gui_qt.py (hunk 2 above)
Comment (lindlou) [reply rpyscope/gui_qt.py:39]: Ok I'll go for `AUTHORS.md`

File: AUTHORS.md [hunk 4]
```diff
@@ -0,0 +1 @@
+This program was started by Reto Trappitsch and completed by Louis Linder
```
[the remark this answers is not in the document]
Comment (lindlou) [reply AUTHORS.md:1]: Hahahah

**Review by trappitsch — APPROVED** [review]

**Comment by trappitsch** [comment]
:ship:

**Comment by lindlou** [comment]
Hurray :D
